example/HelloWorldFunction/build.gradle

```diff
@@ -15,8 +16,11 @@ dependencies {
     aspect 'software.amazon.lambda:powertools-parameters:1.10.3'
     aspect 'software.amazon.lambda:powertools-validation:1.10.3'
 
+    implementation 'software.amazon.lambda:powertools-idempotency:1.10.2'
```

Comment:
Should be 1.10.3 now ? We will anyways upgrade to a feature version before releasing but if not all modules are on same version that will break release draft automation. So please upgrade new modules to latest version of powertools.